src/nevermined/Nfts.ts

```diff
@@ -512,4 +513,102 @@ export class Nfts extends Instantiable {
             throw Error(`Error listing ${ddo.shortId()} on secondary markets`)
         }
     }
+
+    public async buySecondaryMarketNft(
+        buyer: string,
+        seller: string,
+        nftPrice: number,
+        numOfNfts: number = 1,
+        ddo: DDO
+    ) {
+        const buyersAccount = new Account(buyer)
+        const sellersAccount = new Account(seller)
+        const { token } = this.nevermined.keeper
```

Comment:
This would restrict the usability to the NVM token. Ideally we should get the token in params with the price